server/gokbg3/src/main/groovy/com/k_int/ClassUtils.groovy

```diff
@@ -152,7 +152,7 @@ class ClassUtils {
         v = RefdataCategory.lookup(cat, value)
       }
     }
-    else if (value && cat) {
+    else if (value instanceof Integer && cat) {
       try {
         def candidate = RefdataValue.get(value)
```

Comment:
Not sure, but afaik get() works on Database ids of type Long, not Integer.

Reply:
It does work with both actually. GORM defines the type as `java.io.Serializable` internally.